cmd/integrationArtifactDeploy.go

```diff
@@ -99,7 +99,11 @@ func runIntegrationArtifactDeploy(config *integrationArtifactDeployOptions, tele
 		log.Entry().
 			WithField("IntegrationFlowID", config.IntegrationFlowID).
 			Info("successfully deployed into CPI runtime")
-		deploymentError := pollIFlowDeploymentStatus(retryCount, config, httpClient, serviceKey.OAuth.Host)
+		taskId, readErr := ioutil.ReadAll(deployResp.Body)
+		if readErr != nil {
+			return errors.Wrap(readErr, "Task Id not found: HTTP response body could not be read")
```

Comment:
```suggestion
			return errors.Wrap(readErr, "Task Id not found. HTTP response body could not be read.")
```

Okay I know I already suggested a change to this, but I think now that a `.` makes more sense than a `:`